mne/channels/channels.py

```diff
@@ -2146,10 +2244,12 @@ def make_1020_channel_selections(info, midline="z", *, return_ch_names=False):
         # (y-coordinate of the position info in the layout)
         selections = {
             selection: np.array(picks)[pos[picks, 1].argsort()]
+
             for selection, picks in selections.items()
         }
 
     # convert channel indices to names if requested
+
```

Comment:
There are a ton of these newlines, suggesting perhaps your editor is enforcing some style that we don't. (We just use plain `black`.) Can you revert them?

Reply:
Yes sorry I have automated formatting and then I fell into the rabbit hole in git... Please discard this pull request, I am submitting another one cleaner that does exactly the same thing. Sorry for the confusion